src/forms/FormFieldTextArea/FormFieldTextArea.tsx

```diff
@@ -22,7 +22,7 @@ const TextArea = (
 	return (
 		<StyledTextArea
 			id={fieldDef.name}
-			value={value}
+			value={value ?? ""}

```

Comment:
It seems this isn't using the latest code because we deleted all of this component last week